cypress/e2e/views/tags.view.ts

```diff
@@ -18,14 +18,10 @@ export const createTagCategoryButton = "Create tag category";
 export const colorHelper = "div[id=color-helper]";
 export const dropdownMenuToggle = "button[id='tag-type-select-toggle']";
 export const dropdownMenuTypeToggle = "#type-select-toggle";
-export const tagCategoryNameHelper = "div[id=name-helper]";
-export const tagNameHelper = "#-helper";
+export const tagsHelper = "div.pf-v5-c-helper-text";
```

Comment:
The helper selector looks common for tags and stakeholders and other classes it seems .

WE can create one in common view and use everywhere .